### ISSUE TITLE
Arabic letters

### ISSUE THREAD

**Issue by RhodesPeter** [skibinska/romanizer#2]
I was a little confused when it says 'arabic' numbers. I would suggest removing 'arabic'. Otherwise it looks great. 

We could give an example of one test but as they have been using qUnit all day it is probably fine.

**Comment by jsms90** [comment]
@RhodesPeter How about just having a link to [the wiki article](https://en.wikipedia.org/wiki/Arabic_numerals) instead?

The roman numerals kata is a real industry standard -type kata. Especially for learning TDD. It's probably good for anyone using this workshop to know that 0123456789 are "arabic" numerals. If nothing else, because they're likely to hear it again in the future. You don't want to be confused by this for the first time in a tech test.

**Comment by RhodesPeter** [comment]
Thanks @jsms90 - This issue should have been closed as the 'arabic' has already been taken out. 

I feel that calling numbers 'arabic numerals' in the workshop is not needed and simply complicates the instructions in a way that doesn't add the learning outcomes. 

That said, I don't feel too strongly and linking to the wiki would certainly be the easiest way to alleviate the confusion (that I felt).

**Comment by jsms90** [comment]
Yeah, I only think its useful because it is generally used everywhere else when this kata is done.

Examples from the in a google search for "roman numerals kata":
http://agilekatas.co.uk/katas/RomanNumerals-Kata
https://dzone.com/articles/roman-numerals-kata-tdd-and
https://remonsinnema.com/2011/12/05/practicing-tdd-using-the-roman-numerals-kata/
http://geekswithblogs.net/theArchitectsNapkin/archive/2014/05/28/informed-tdd-ndash-kata-ldquoto-roman-numeralsrdquo.aspx

I reckon the best thing we can do is expose students to anything that might confuse them later, in a nicer learning environment first.

But maybe the point isn't worth mentioning now unless someone actually felt strongly enough to go and put it back in :sweat_smile:
